.gitignore

```diff
@@ -1,6 +1,5 @@
 *.a
-opencog.log
-opencog-python.log
+*.log
```

Comment:
It would be nice if we stopped writing out forward-chainer.log. Its always huge, and it pollutes the filesystem.